website/content/docs/connect/manage-traffic/route-to-local-upstreams.mdx

```diff
@@ -49,11 +51,11 @@ locality = {
 }
 ```
 
-## Specify the localities of your service instances
+## Specify the localities of your service instances (optional)
```

Comment:
We talked previously about removing this section entirely. I'm fine w/ that if we think it's not useful to document this use case explicitly, but I also don't think it's harmful to include if we call out it's not typical.